introduce DOCBUILDER_ARGUMENTS env var to provided args for genmaindoc

ngoan1608 opened the pull request:
Hi Holger,
Hi Thomas,

The argument `--configfile` is only used with **genmaindoc.py** script and this script is called indirectly (without additional argument) by **setup.py**.

I think passing this argument from **build** => **setup.py** =>**genmaindoc.py** is not a good idea, it is also confused because **build** has `--config-file` for repos configuration.

So that, I add the environment variable `DOCBUILDER_ARGUMENTS` so that you can specify the additional argument (such as config file) for building main document.
Thi env var is only configured for Gitlab CI to use the config file located on Gitlab **build** repo.

Please review and give your idea about it,

Thank you,
Ngoan

ngoan1608 committed:
introduce DOCBUILDER_ARGUMENTS env var to provided args for genmaindoc

HolQue commented:
Hello Ngoan,

your solution is fine to me. Please update also version and date in repository_config.json.

ngoan1608 committed:
update README and versions accordingly

ngoan1608 commented:
> Hello Ngoan,
> 
> your solution is fine to me. Please update also version and date in repository_config.json.

Hi Holger,

Thank you for your reminder.
I have updated version information accordingly.
Besides, README is also updated to provide the information about this environment variable.

Thank you,
Ngoan

test-fullautomation reviewed (approved):
Hi Ngoan,
looks good to me!
Thank you,
 Thomas
